cafebabel/articles/translations/views.py

```diff
@@ -23,7 +23,7 @@ def create():
                 original_article=article.id,
                 status='draft',
                 editor=article.editor,
-                author=article.author,
+                authors=[article.author],
```

Comment:
Why not `article.authors`?

Reply:
That makes me wonder what should happen when updating the list of the original article's authors.
I suppose the translations should follow.